### PR TITLE
Add Smartcity Dashboard Use Case

### PR DESCRIPTION
**Pull request by mmccool**
Addresses Issue https://github.com/w3c/wot-usecases/issues/17

### CODE REVIEW

File: USE-CASES/smartcity-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+The user, a member of city management responsible for making planning decisions,
+sees data visualized on a map suitable for planning decisions.
+
+#### Variants:
+
+* Historical data may also be available (allowing an analysis of trends over time).
+* It may be possible to also issue commands to actuators through the interface.
+* The system may be used for emergency response (for instance, closing floodgates in response to an expected tsunami)
+* A subset of the data visualization capabilties may be made available to the public (for example, traffic)
+
```
Comment (jenlincode) [inline USE-CASES/smartcity-dashboard.md:99]: Add: 

Filtering based off parameters, location (area, state, county, country, zip code?), sensor type, subject matter, etc.

Comment (jenlincode) [inline USE-CASES/smartcity-dashboard.md:99]: Ability to generate alerts off of various parameters.

Comment (jenlincode) [inline USE-CASES/smartcity-dashboard.md:99]: Ability to produce logs off historical data.

**Comment by mlagally** [comment]
UC call 12.1.:
Decided to merge the current form.
Updates to incorporate Jennifer's feedback will be done on the HTML version.

**Pull request merged**